File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/Java/Pattern.java
class Pattern{
	
	public static void main(String args[]){
		
		int num = 1;

		for(int outerItr = 0 ; outerItr < 4 ; outerItr++){

			for(int innerItr = 0 ; innerItr <= outerItr ; innerItr++){
				System.out.print(num*num+"\t");
				num++;
			}
			System.out.printf("\n");


		}	
	}	
}   

File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/C/pattern.c

#include<stdio.h>

void main(){

	int num = 1;
	for(int outerItr = 0 ; outerItr < 4 ; outerItr++){

		for(int innerItr = 0 ; innerItr <= outerItr ; innerItr++){
			printf("%d\t",num*num);
			num=num+1;
		}	
		printf("\n");


	}	
}	   
